examples/xla_step.py

```diff
@@ -35,14 +39,17 @@ def gym_sync_step() -> None:
   def actor_step(iter, loop_var):
     handle0, states = loop_var
     action = policy(states)
-    handle1, (new_states, rew, done, info) = step(handle0, action)
+    if is_legacy_gym:
+      handle1, (new_states, rew, done, info) = step(handle0, action)
+    else:
+      handle1, (new_states, rew, term, trunc, info) = step(handle0, action)
     return (handle1, new_states)
 
   @jit
   def run_actor_loop(num_steps, init_var):
     return lax.fori_loop(0, num_steps, actor_step, init_var)
 
-  states = env.reset()
+  states, _ = env.reset()
```

Comment:
```suggestion
  if is_legacy_gym:
    states = env.reset()
  else:
    states, _ = env.reset()
```
same as other places

Reply:
Sorry, I missed those. Updated.